cfme/tests/configure/test_docs.py

```diff
@@ -67,6 +67,20 @@ def docs_info():
             'User',
             'Settings and Operations'
         ]
+    elif version.appliance_is_downstream():
+        return [
+            'Monitoring Alerts Reporting',
+            'General Configuration',
+            'Virtual Machines Hosts',
+            'Methods For Automation',
+            'Infrastructure Inventory',
+            'Providers',
+            'Scripting Actions',
+            'Defining Policies Profiles'
+        ]
+    else:
+        # Upstream version has no docs
+        return []
```

Comment:
This test also verifies that About page can be opened (though it might be verified by a different test). Time wasted is not really huge - around 1 min.

See also https://github.com/ManageIQ/manageiq/issues/2902